### DISCUSSION TITLE
Dungeon browser disabled in TBC until later phases

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#3924] [Suggestions]
Turning RDF off for TBC dungeons and releasing it only partially/progressively.

One concept could be:
- 61-69 -> no TBC rdf normal mode dungeons
- with release of 70 or t4: rdf for TBC normal mode dungeons
- with the bracket afterwards: rdf for TBC heroic mode dungeons.

Blizzard itself realized that the dungeon browser was no smart decision and will not bring it back for Wrath Classic.
Considering, we could just disable TBC dungeons (normal and heroic), would bring back the retail tbc experience.
Classic dungeons could still be left accessible through the rdf for everyone leveling etc.

Pros:
- people would explore the world more, maybe even spam less rdfs and thus, we would receive more bugreports for the open world.
- less worrying about rdf satchel loot

**Comment by heyitsbench** [comment]
Gonna have to disagree. A couple cons are that a lack of RDF *requires* high population to work well. RDF throughout the level phases of CC isn't exactly instant, and without RDF, dungeon testing wouldn't be high enough to justify it. The TBC 61-69 phase is also split in half, but the dungeon count is still as high or higher as any other phase in the past for each half, and having people spam the world chat for a specific dungeon for each of the fifteen dungeons in the phase is far from ideal.

**Reply by balleny** [reply]
and from my past experience, i can disagree to your statement.
not having rdf works just fine, also on lower pop realms.
People will run dungeons for the questrewards, achievements, loot and pre-quests to karazhan on mass. TBC dungeons will be in demand and people will group up / form groups via LFM chat just fine

**Comment by BeefSteaker** [comment]
Please no. Rdf is a big QoL addition.

Blizzard not adding it to wotlk classic is hardly a good argument as it's very controversial
